src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
do you need these properties for the card? block elements are aligned vertically by default